code/extensions/che-telemetry/src/extension.ts

```diff
@@ -0,0 +1,60 @@
+/**********************************************************************
+ * Copyright (c) 2022 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ***********************************************************************/
+import * as vscode from 'vscode';
+import { TelemetryEventService } from "./telemetry-event-service";
+
+let telemetryEventService: TelemetryEventService;
```

Comment:
Is there a reason to move the `telemetryEventService` outside of the `activate` function?
Can we declare `telemetryEventService` as a constant within the `activate`?

Reply:
Fixed, moved to be inside the `activate` function